bibliopixel/drivers/ledtype.py

```diff
@@ -0,0 +1,25 @@
+from enum import IntEnum
+
+
+class LEDTYPE(IntEnum):
+    GENERIC = 0  # Use if the serial device only supports one chipset
+    LPD8806 = 1
+    WS2801 = 2
+    # These are all the same
+    WS2811 = 3
```

Comment:
I should say that i never loved this idea of having two different enums being "the same value" but different names because "they work the same".  But that ain't changing in 3.x!